test/testcafe/spec/WidgetCustomization_spec.js

```diff
@@ -129,7 +132,7 @@ test.requestHooks(xhrSelectAuthenticatorMock)('can customize back to signin link
   await t.expect(selectAuthenticatorPageObject.getSignoutLinkText()).eql('Back to sign in');
 });
 
-test.requestHooks(identifyMock)('should show custom buttons links', async t => {
+test.meta('v3', false).requestHooks(identifyMock)('should show custom buttons links', async t => {
```

Comment:
Custom buttons are out of scope

Reply:
Can you add a comment above this stating that custom buttons are not supported yet, with a JIRA reference to the ticket that will add support?

Reply:
this is the custom buttons ticket: https://oktainc.atlassian.net/browse/OKTA-594754

Reply:
Done